xfce/libxfce4ui/Portfile

```diff
@@ -3,11 +3,11 @@
 PortSystem 1.0
 
 name            libxfce4ui
-version         4.16.1
+version         4.18.6
```

Comment:
Please update to the current version.

Reply:
As far as I understand, 4.19.x versions are development versions for what will ultimately become the 4.20.x series.